packages/shared/styles/components/atoms/SfChevron.scss

```
@@ -2,32 +2,64 @@
 
 .sf-chevron {
   position: relative;
-  width: var(--chevron-size, 1.25rem);
-  height: var(--chevron-size, 1.25rem);
+  width: var(--chevron-width, var(--chevron-size, 1.25rem));
+  height: var(--chevron-height, var(--chevron-size, 1.25rem));
   cursor: pointer;
   &__bar {
     position: absolute;
     top: 50%;
-    background: transparent;
-    transition: all 500ms cubic-bezier(0.25, 1.7, 0.35, 0.8);
+    background: var(--chevron-background, transparent);
+    transition: all 300ms cubic-bezier(0.25, 1.7, 0.35, 0.8);
```

```suggestion
    transition: transform 300ms cubic-bezier(0.25, 1.7, 0.35, 0.8);
```
`all` is a not good idea :)